CollAction/Services/Project/IParticipantsService.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Threading.Tasks;
+using CollAction.Models;
+
+namespace CollAction.Services.Project
+{
+    public interface IParticipantsService
+    {
+        Task<AddParticipantResult> AddAnonymousParticipant(int projectId, string email);
+        Task<AddParticipantResult> AddLoggedInParticipant(int projectId, string userId);
+        Task RefreshParticipantCountMaterializedView();
+        Task<ProjectParticipant> GetParticipant(string userId, int projectId);
+        Task<string> GenerateParticipantsDataExport(int projectId);
+    }
+
+    public enum AddParticipantScenario
```

Comment:
I think the guideline is that separate interfaces/enums/classes should be in separate files

Reply:
Done